doc/lua.yaml

```diff
@@ -184,3 +184,44 @@ values:
+      | font                | The name of the font to be used, `Fontconfig` is used to search for the font.        |
+      | font_size           | The font size.                                                                       |
+      | alignment           | One of `CAIRO_TEXT_ALIGN_LEFT`, `CAIRO_TEXT_ALIGN_RIGHT`, `CAIRO_TEXT_ALIGN_CENTER`. |
+      |                     | Default value: `CAIRO_TEXT_ALIGN_LEFT`                                               |
+      | language            | A string containing a BCP 47 language tag.                                           |
+      |                     | Default value: `en`                                                                  |
+      | script              | A string containing a ISO 15924 script tag.                                          |
+      |                     | Default value: auto-detect from text                                                 |
+      | direction           | A string representing text direction eg `LTR`, `RTL` or `TTB`                        |
+      |                     | Default value: auto-detect from text otherwise `LTR`                                 |                                                               |
+  - name: width,height:cairo_text_hp_text_size(text, font, font_size, language, script, direction)
+    desc: |-
+      Used to calculate how many pixels will be required to render a string with
+      `cairo_text_hp_show` returns a pair of int's with `width` and `height`.
+
+      | Argument            | Description                                                                          |
+      |---------------------|--------------------------------------------------------------------------------------|
+      | cr                  | The `cairo_t` to render to.                                                          |
+      | text                | The text to render.                                                                  |
+      | font                | The name of the font to be used, `Fontconfig` is used to search for the font.        |
+      | font_size           | The font size.                                                                       |
+      | language            | A string containing a BCP 47 language tag.                                           |
+      |                     | Default value: `en`                                                                  |
+      | script              | A string containing a ISO 15924 script tag.                                          |
+      |                     | Default value: auto-detect from text                                                 |
+      | direction           | A string representing text direction eg `LTR`, `RTL` or `TTB`                        |
+      |                     | Default value: auto-detect from text otherwise `LTR`                                 |
```

Comment:
Instead of putting default value into the next row, it might be better to put it in the same cell and use `<br/>` so that it looks like this:

| Argument | Description |
|-|-|
| language | A string containing a BCP 47 language tag. <br/>  Default value: `en` |

Reply:
This also should be fixed.